src/stirling/source_connectors/socket_tracer/BUILD.bazel

```diff
@@ -546,7 +540,7 @@ pl_cc_bpf_test(
 
 pl_cc_bpf_test(
     name = "go_tls_trace_bpf_test",
-    timeout = "moderate",
+    timeout = "long",
```

Comment:
While running builds against #2115, I noticed that this test is flaky after merging #2116. This is because we added 2 go versions and only removed one ([diff](https://github.com/pixie-io/pixie/pull/2116/files#diff-4c54c2ecc1c0bf68208d05a35c85cafc82eec10c800f0d7e0c60397cbaf665cb)). Increasing the timeout should solve that problem.

Reply:
Given that `go1.24` was recently released and the golang policy is to old support upto two versions too old, I think. you can drop `1.18`. I'd probably also suggest dropping `1.19` too.
IMO increasing the test matrix too much doesn't seem worth it.

Reply:
Yea, I agree that we should deprecate those soon.

Omid had concerns that there might be older Go versions in use by end users despite Go's policy. I know of a few end users in this situation, so I might consider how to keep that test coverage without having our own dependencies tied to it.

If I decide to abandon keeping that coverage, at least its much easier to drop old support now with the dynamic logging stuff figured out. So a subsequent removal will be much easier.